Example/App.js

```diff
@@ -48,7 +60,7 @@ const App = () => {
     prefetchConfiguration({
       warmAndPrefetchChrome: true,
       connectionTimeoutSeconds: 5,
-      ...configs.identityserver,
+      ...configs.auth0,
```

Comment:
Note! the demo.identityserver has an expired cert, so this prefetch was actually failing and not surfacing the error to JS – it'd just hang on a white screen. Probably need to look into that at some point, but for now I just changed this to prefetch auth0 config.